cmd/scollector/doc.go

```diff
@@ -278,6 +278,22 @@ metrics.
 
 	LocalListener = "localhost:4242"
 
+Oracles (array of table, keys are ClusterName, Instances): Oracle database
+instances to poll. The Instances key is an array of table with keys
+ConnectionString and Role, which are the same as using sqlplus.
+
+	[[Oracles]]
+	  ClusterName = "oracle rac name"
+	  [[Oracles.instances]]
+	    ConnectionString = "/"
```

Comment:
these should be ConnectString not ConnectionString, or the struct should be changed